IntroOOP/Task2/Rectangle.cs

```diff
@@ -0,0 +1,38 @@
+﻿namespace IntroOOP.Task2;
+
+public class Rectangle : Point
+{
+    #region FIELDS
+    private int _a, _b;
+
+    private double _area, _perimetr;
+
+    #endregion
+
+    public Rectangle(string color, bool state, int x, int y, int a, int b) : base(color, state, x, y)
+    {
```

Comment:
```suggestion
    public Rectangle(
        string color, 
        bool state, 
        int x, 
        int y, 
        int a, 
        int b) 
        : base(color, state, x, y)
    {
```
Когда много параметров лучше смотрится их запись в столбик.